test/Microsoft.Azure.WebJobs.Extensions.ServiceBus.Tests/ServiceBusEndToEndTests.cs

```diff
@@ -225,19 +260,25 @@ private async Task<int> CleanUpEntity(string queueName, string connectionString
                     break;
                 }
             } while (true);
+
             await messageReceiver.CloseAsync();
+
             return count;
         }
 
         private async Task Cleanup()
         {
-            await CleanUpEntity(FirstQueueName);
-            await CleanUpEntity(SecondQueueName);
-            await CleanUpEntity(BinderQueueName);
-            await CleanUpEntity(FirstQueueName, _secondaryConnectionString);
-
-            await CleanUpEntity(EntityNameHelper.FormatSubscriptionPath(TopicName, TopicSubscriptionName1));
-            await CleanUpEntity(EntityNameHelper.FormatSubscriptionPath(TopicName, TopicSubscriptionName2));
+            var tasks = new List<Task>
+            {
+                CleanUpEntity(FirstQueueName),
```

Comment:
These tests take way too long to run. At least doing these in parallel will help. We should really refactor most of these tests to be direct invoke (JobHost.CallAsync), since that allows us to test binding permutations quickly. We still need some E2E tests though that actually pull messages, but those can be much fewer.